### PR TITLE
Command pallete polish

### PR DESCRIPTION
**Pull request by Ngoakor12**
Pull-Request for `paretOS`

## Description
- Fix page navigation
- remove name duplicate

## Relates to
- #260 

## Screenshots / other info
...

### CODE REVIEW

File: src/containers/Palette.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Button, Dialog } from "@mui/material";
 import "ninja-keys";
 import sanity from "../libs/sanity";
 import ExternalSiteModal from "../context/ExternalSiteModal";
-
+import { useHistory } from "react-router-dom"
```
Comment (jeremydthomas) [inline src/containers/Palette.tsx:7]: As of react router dom 6 useHistory is deprecated for useNavigate

**Review by jeremydthomas — COMMENTED** [review]
As of react router dom 6, useHistory is deprecated for useNavigate. 

[docs](https://reactrouter.com/en/6.4.1/hooks/use-navigate) if you are interested.

**Comment by jeremydthomas** [comment]
Good job though, RR6 is quite a bit different than 5!

**Comment by Ngoakor12** [comment]
> Good job though, RR6 is quite a bit different than 5!

Thanks for the review. I knew RR6 had many changes, but these ones were not on my radar. Will have a look and make changes now now.

**Comment by Ngoakor12** [comment]
Also @jeremydthomas , the `Routes.tsx` seems has some errors, related to the whole RR6, My PR was does not need it to work, I just thought I should raise it.

Should I try and solve it as part of this PR? As part of another PR maybe? Or leave it to you?

**Comment by jeremydthomas** [comment]
> 

I can't open your picture on my phone for some reason, I'll get on my computer later and try.

**Comment by Ngoakor12** [comment]
> I can't open your picture on my phone for some reason, I'll get on my computer later and try.
> still cant open it, not sure whats wrong.

It wasn't showing on my side aswell, but I updated, you should be able to see it now
> offtopic, but this command pallet is cool, pretty easy to navigate around that way

Yep, couldn't agree more, very convenient especially if you like navigating via a keyboard
> Im not seeing any obvious errors, even when Im in your branch. Did you make sure you deleted node_modules and reinstall everything? I changed quite a bit of packages that are deprecated.

I think I did everything well, had to delete node modules a couple of times and run `pnpm install` and get it setup correctly because my fork was out of sync and didn't have the latest RR6 merge, but I was able to sync everything I think it's fine now. Have a look at the screenshot and let me know what you think, I can try and reinstall the packages one more time.

**Comment by jeremydthomas** [comment]
I'll try looking at this today, I'm on day shift for 4 more days and we are a bit busy.

**Comment by Ngoakor12** [comment]
> I'll try looking at this today, I'm on day shift for 4 more days and we are a bit busy.

No problem, thanks for letting me know :thumbs_up:

**Comment by jeremydthomas** [comment]
> Also @jeremydthomas , the `Routes.tsx` seems has some errors, related to the whole RR6, My PR was does not need it to work, I just thought I should raise it. Screenshot from 2022-10-02 21-12-46
> 
> Should I try and solve it as part of this PR? As part of another PR maybe? Or leave it to you?

I see what I did, after further looking, the exact keyword is no longer supported in RR6, Ill remove them. There is some typing that needs to be fixed though.

**Review by jeremydthomas — APPROVED** [review]

**Comment by mikhael28** [comment]
Great work you two!

**Pull request merged**